admin.list

```diff
@@ -0,0 +1,32 @@
+scripts/2.6.0: Do not load a specific version of the scripts module. Only do: module load scripts
+
+scripts/3.0.0: Do not load a specific version of the scripts module. Only do: module load scripts
+
+scripts/3.2.1: Do not load a specific version of the scripts module. Only do: module load scripts
+
+scripts/4.0.0: Do not load a specific version of the scripts module. Only do: module load scripts
```

Comment:
although we do need to make sure we don't start nagging about `scripts/5.0.0` I guess...